src/Metadata/AmazonMetadataBuilder.php

```diff
@@ -48,12 +73,20 @@ final class AmazonMetadataBuilder implements MetadataBuilderInterface
     ];
 
     /**
-     * @var MimeTypesInterface
+     * @var array
+     *
+     * @phpstan-var array{
+     *     'standard': self::STORAGE_STANDARD,
+     *     'reduced': self::STORAGE_REDUCED,
+     * }
      */
-    private $mimeTypes;
+    private $storage = [
```

Comment:
What's the purpose of the private property if you can't change the values ?

Reply:
Done